package.json

```diff
@@ -44,63 +45,54 @@
     "launchdarkly-js-client-sdk": "^2.24.2",
     "lodash": "^4.17.21",
     "lodash.omit": "^4.5.0",
-    "pdfjs-dist": "3.4.120",
+    "pdfjs-dist": "3.9.179",
     "pinia": "^2.0.35",
     "pinia-class": "^0.0.3",
     "regenerator-runtime": "^0.13.11",
     "sbc-common-components": "3.0.8",
     "tiptap-vuetify": "^2.24.0",
     "uuid": "^9.0.0",
     "v-sanitize": "^0.0.11",
-    "vue": "2.7.10",
+    "vue": "2.7.14",
     "vue-affix": "^0.5.2",
-    "vue-debounce-decorator": "^1.0.1",
     "vue-hotjar": "^1.4.0",
     "vue-router": "^3.6.5",
     "vue-the-mask": "^0.11.1",
     "vue2-filters": "^0.14.0",
-    "vuelidate": "^0.7.7",
+    "vuelidate": "0.6.2",
     "vuetify": "^2.6.15",
     "vuex": "^3.6.2"
   },
   "devDependencies": {
     "@babel/plugin-proposal-private-methods": "^7.18.6",
+    "@esbuild-plugins/node-modules-polyfill": "0.2.2",
```

Comment:
Does Create UI need this one?

Is "@originjs/vite-plugin-require-context" also needed here?

Reply:
No, it doesn't. Here's the thing Sev:
As each UI is a little bit different that the other, there's a special configuration for each. For example, this package here is needed since Edit UI uses TipTapVuetify while Create UI doesn't.

As for the `@originjs/vite-plugin-require-context` package, no we don't need it since in Create UI, there is a usage of `require.context(xxx)` while here there isn't any.

Reply:
Got it. I wish we could add code comments somewhere explaining this... maybe where TipTapVuetify is imported or require is used?

Reply:
I've actually already added one! https://github.com/JazzarKarim/business-edit-ui/blob/e411b689e10afae95236e57a1b75c009cb1e7258/vite.config.js#L40

Reply:
Ah, I noticed that but didn't connect it to the "node-modules-polyfill" package.

Reply:
I'm sorry, this package is actually to fix  a browser compatibility warning not tiptapvuetify:
https://github.com/JazzarKarim/business-edit-ui/blob/e411b689e10afae95236e57a1b75c009cb1e7258/vite.config.js#L64

Sorry Sev.